.github/workflows/sycl-containers.yaml

```diff
@@ -68,7 +68,7 @@ jobs:
           username: ${{ github.repository_owner }}
           password: ${{ secrets.GITHUB_TOKEN }}
           tags: |
-            ghcr.io/${{ github.repository }}/${{ matrix.tags }}-${{ github.sha }}
-            ghcr.io/${{ github.repository }}/${{ matrix.tags }}:latest
+            ghcr.io/${{ github.repository }}/${{ matrix.file }}-${{ github.sha }}
```

Comment:
Actually, we probably need to differentiate between devigc, latest and unstable images. Here all three will use the same tag. We might need to use `tags` parameter to build the tag name as well.

Reply:
Good catch, fixed.